pkg/controller/fileintegrity/config_defaults.go

```diff
@@ -46,158 +46,24 @@ CONTENT_EX = sha512+ftype+p+u+g+n+acl+selinux+xattrs
 DATAONLY =  p+n+u+g+s+acl+selinux+xattrs+sha512
 
 /hostroot/boot/        CONTENT_EX
-/hostroot/opt/        CONTENT
 /hostroot/root/\..* PERMS
 /hostroot/root/   CONTENT_EX
+!/hostroot/root/.xauth*
```

Comment:
Do we want this? probably RHCOS won't have X installed

Reply:
I'll check, but I left that in from the default. Doesn't hurt if its there though.